lib/config/genconfig/hadeployconfig.go

```diff
@@ -1064,49 +1064,60 @@ func (c *HaDeployConfigGen) PromptBackup() (err error) {
 
 func (c *HaDeployConfigGen) PromptObjectStorageSettings(backupOption string) (err error) {
 
-	bucketName, err := c.Prompt.InputStringRegexErrMsg("Bucket name", BUCKET_NAME_REGEX, ERR_MSG_BUCKET)
+	bucketName, err := c.Prompt.InputStringRegex("Bucket name", BUCKET_NAME_REGEX)
```

Comment:
```suggestion
	bucketName, err := c.Prompt.InputStringRegexErrMsg("Bucket name", BUCKET_NAME_REGEX, ERR_MSG_BUCKET)
```
This was the change to provide user friendly error message for Regex checks.